docs/changelog.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :bug:`5050` Provide a fix for the edge case at 1.25.3->1.26.0 (v34->v35) DB upgrade that caused a FOREIGN key error and botched the upgrade.
+* :bug:`-` Assets from bitcoin.de have been correctly mapped.
```

Comment:
This is not a proper user facing changelog. A user does not understand what this means.

```suggestion
* :bug:`-` Having SOL in custody in bitcoin.de will now work properly again.
```